docs/config.md

```diff
@@ -14,17 +14,19 @@ USAGE
   $ symbol-bootstrap config
 
 OPTIONS
-  -a, --assembly=assembly                   The assembly, example "dual" for testnet. If not provided, the value is
-                                            resolved from the target/preset.yml file.
+  -a, --assembly=assembly                   The assembly that define the node(s) layout. It can be provided via custom
```

Comment:
that define*s

Reply:
all three docs files. 

as there is a lot of similarity, i was wondering: are you generating these manually or automatically?

Reply:
Fixed. They are automatically generated

Reply:
if you don't already, consider adding a build section to the readme (if you don't already) about how to generate these files and the expected test reporting files too. can be separate PR.